packages/expo-store-review/android/src/main/java/expo/modules/storereview/StoreReviewModule.kt

```diff
@@ -53,7 +53,7 @@ class StoreReviewModule(private val mContext: Context)
           }
         }
       } else {
-        promise.reject(null)
+        promise.reject("ERR_STORE_REVIEW_FAILED", "Android ReviewManager flow failed")
```

Comment:
```suggestion
        promise.reject("E_STORE_REVIEW_FAILED", "Android ReviewManager flow failed")
```

Reply:
Just curious about the intended convention here. The other error path uses code `ERR_STORE_REVIEW_FAILED`. Should I change both to `E_STORE_REVIEW_FAILED`, or is the difference intentional?

Reply:
Looks like you're right, I think the `E_` prefix was abandoned.